ratis-server/src/test/java/org/apache/ratis/server/impl/MiniRaftCluster.java

```diff
@@ -410,11 +441,21 @@ public PeerChanges addNewPeers(int number, boolean startNewPeer)
 
   public PeerChanges addNewPeers(int number, boolean startNewPeer,
       boolean emptyPeer) throws IOException {
-    return addNewPeers(generateIds(number, servers.size()), startNewPeer, emptyPeer);
+    return addNewPeers(generateIds(number, servers.size()), startNewPeer, emptyPeer, false);
   }
 
   public PeerChanges addNewPeers(String[] ids, boolean startNewPeer,
       boolean emptyPeer) throws IOException {
+    return addNewPeers(ids, startNewPeer, emptyPeer, false);
+  }
+
+  public PeerChanges addNewPeers(int number, boolean startNewPeer,
+      boolean emptyPeer, boolean isListener) throws IOException {
```

Comment:
Similar to other code, pass RaftPeerRole instead of isListener.

Reply:
done